authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -342,6 +342,13 @@ public SecurityFilterChain oidcWebappSecurity(
         throws Exception {
       return httpSecurity
           .securityMatcher(WEBAPP_PATHS.toArray(new String[0]))
+          .authorizeHttpRequests(
+              (authorizeHttpRequests) ->
+                  authorizeHttpRequests
+                      .requestMatchers(UNPROTECTED_PATHS.toArray(String[]::new))
```

Comment:
:question: Didn't we also talk about the need to add more paths, for example `/identiy/assets`?

Reply:
In OIDC mode we don't need to add any more paths because in theory the UI should be available only when you have access (for example the access checks also rely on an authenticated user) the login page is external to us so this should be fine too